src/pages/Folder/components/FolderCard/index.jsx

```diff
@@ -0,0 +1,96 @@
+import styles from "./styles.module.css";
+import { getTimeAgo, formatDate } from "util/time";
+import noImg from "assets/noImg.png";
+import starImg from "assets/star.png";
+import kebabImg from "assets/kebab.png";
+import { useState } from "react";
+import { ModalPortal, PopOver, DeleteModal, AddLinkModal } from "components";
+import { useModal } from "hooks/useModal";
+
+export function FolderCard({ link, folders }) {
+  const createdAt = "createdAt" in link ? "createdAt" : "created_at";
+  const imageSource = "imageSource" in link ? "imageSource" : "image_source";
+
+  const [openPopOver, setOpenPopOver] = useState(false);
+
+  const { openModal, modalRef, handleModalClose, handleModalOpen } = useModal();
+  const {
+    openModal: addOpenModal,
+    modalRef: addModalRef,
+    handleModalClose: addHandleModalClose,
+    handleModalOpen: addHandleModalOpen,
+  } = useModal();
+
+  const handlePopOverClose = () => {
+    setOpenPopOver(false);
+  };
+
+  const handleMoreBtn = (e) => {
+    //a태그 페이지 이동 막기
+    e.preventDefault();
+    if (openPopOver) {
+      return;
+    }
```

Comment:
P3:
early return문의 경우 제 개인 취향으로는 아래와 같은 방식이 더 가독성이 좋다고 생각합니다.
```suggestion
    if (openPopOver) return;
```